src/Compilers/CSharp/Test/Symbol/SymbolDisplay/SymbolDisplayTests.cs

```diff
@@ -2869,6 +2869,22 @@ public void TestMultipleCustomModifier()
             Verify(description, expectedText, expectedKinds);
         }
 
+        private static void TestSymbolDescription(
```

Comment:
Can previous function delegate to this one? So that we reuse the code rather than duplicate it.